oioioi/base/templatetags/format_data_range.py

```diff
@@ -0,0 +1,20 @@
+from django import template
+
+register = template.Library()
+
+@register.simple_tag
+def format_data_range(start, end):
```

Comment:
Currently the %b is returning month name according to server locale (usually EN).
So even with polish language selected in user profile it would return sth like:
```(04 Oct 2023, 12:03 - 04 Oct 2099, 14:03)```

I recommend to use sth like:
```
from django.utils import formats

...
        start_str = formats.date_format(start, "j E Y, H:i")
```